specs/protocol/block_header.md

```diff
@@ -0,0 +1,24 @@
+# Block Header
+
+## Application Header
+
+name                 | type       | description
+---------------------|------------|-----------------------------------------------------------------------------------------------------
+`height`             | `uin64`    | Height of this block.
+`timestamp`          | `uin64`    | Time this block was created, in [TAI64](https://cr.yp.to/libtai/tai64.html) format.
+`txCount`            | `uint64`   | Number of [transaction](./tx_format.md#transaction)s in this block.
+`txRoot`             | `byte[32]` | [Merkle root](./cryptographic_primitives.md#binary-merkle-tree) of [transaction](./tx_format.md#transaction)s in this block.
```

Comment:
can we move the `outputMessageCount` to be right after the `txRoot` so we have 4 uint64s in a row? This will help reduce data in the solidity implementation

Reply:
Fixed in [6a91e0b](https://github.com/FuelLabs/fuel-specs/pull/405/commits/6a91e0b01b04db45f2c0a36c1d061c73e37cdbf4)